tests/test_role.py

```diff
@@ -0,0 +1,10 @@
+"""Test suite for restriction of endpoints by Role."""
+
+from .base_test import BaseTestCase
+
+
+class RoleTestCase(BaseTestCase):
+    """Holds tests of roles in authentication."""
+
+    def test_andelan_can_access_(self):
+        pass
```

Comment:
I imagine this is incomplete.

Reply:
@LarryWachira Still quite a lot of confusion due to the attempted rebase of the work.
All these shall be resolved ASAP

Reply:
It has bee completed